### PR TITLE
fix upload image after delete

### PR DESCRIPTION
**Pull request by mostafa-hisham**
fix #[160](https://github.com/Strategy11/business-directory-premium/issues/160)

### CODE REVIEW

**Comment by mostafa-hisham** [comment]
@garretlaxton could you please take another look at it? I fixed the issue

**Review by garretlaxton — APPROVED** [review]
Nice, this looks great now!

File: assets/js/wpbdp.js [hunk 1]
```diff
@@ -275,6 +275,9 @@ WPBDP.fileUpload = {
                         if ( ! t._admin_nonce ) {
                             t._slotsRemaining++;
                             $( '#image-slots-remaining' ).text( t._slotsRemaining );
+							if(t._slotsRemaining < t._slots){
```
Comment (stephywells) [inline assets/js/wpbdp.js:278]: Let's update this line to match the codestyling in other lines. It should be:
`if ( t._slotsRemaining < t._slots ) {`

**Review by stephywells — CHANGES_REQUESTED** [review]
This works great! Let's fix the codestyling here too, and then we can marge it.

**Review by stephywells — APPROVED** [review]
Perfect!

**Pull request merged**